applications/Unity.GrantManager/modules/Volo.BasicTheme/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic/Bundling/BasicThemeGlobalStyleContributor.cs

```diff
@@ -6,11 +6,40 @@ public class BasicThemeGlobalStyleContributor : BundleContributor
 {
     public override void ConfigureBundle(BundleConfigurationContext context)
     {
-        context.Files.Add("/themes/basic/fonts/*/*");
         context.Files.Add("/themes/basic/fluentui-icons.css");
         context.Files.Add("/themes/basic/fluenticons.min.css");
         context.Files.Add("/themes/basic/fonts.css");
         context.Files.Add("/themes/basic/layout.css");
         context.Files.Add("/themes/basic/unity-styles.css");
+
+        // Add assets for "/themes/basic/fonts/**/*"
+        context.Files.AddRange(new[] {
+            "/themes/basic/fonts/icons/Segoe-Fluent-Icons.ttf",
```

Comment:
We do not need to include font files like this, 
fonts were loading correctly using the previous method. 
Home button font loading is a different issue and it's been fixed now(will be In my next pr)

Reply:
<img width="1234" alt="Screenshot 2023-08-28 at 11 15 54 AM" src="https://github.com/bcgov/Unity/assets/60233274/4995394f-525f-4871-b121-e8dc4bce8bcb">

Somehow the font files were loading in the dev, Maybe it's not containerized in the dev?

Let's keep adding files this way if it solves the containerizing issue.